other/buildTools/DCO.txt

```diff
@@ -0,0 +1,25 @@
+Developer's Certificate of Origin 1.1
```

Comment:
Yup, I'll fix these as part of the fixing up the licence stuff, just wanted to get these in and fix the submodule fun first.